frontend/src/infra/MyProvider.js

```diff
@@ -22,14 +21,15 @@ const MyProvider = (props) => {
     joinDate: "",
     roomCreationDate: "",
   });
-
+  let history = useHistory();
   socket.on("NewPlayer", (data) => {
     console.log("New Player from socket");
     setState({ ...state, playerList: data.roomPlayers });
   });
 
   const onNewGame = () => {
     console.log("OnNewGame Pressed");
+
```

Comment:
why so many whitespaces

Reply:
Sometimes whitespaces make it more readable